app/config.py

```diff
@@ -39,6 +46,13 @@ class Config(metaclass=MetaFlaskEnv):
     HTTP_SCHEME = "https"
     FRONTEND_HOSTNAME = None
 
+    REDIS_URL = os.getenv("REDIS_URL")
+    REDIS_ENABLED = REDIS_URL is not None
+
+    # 100 requests every 5 minutes
+    DOCUMENT_AUTHENTICATION_RATE_LIMIT = int(os.getenv("DOCUMENT_AUTHENTICATION_RATE_LIMIT", "100"))
```

Comment:
Just pondering this again actually, is 100 a bit generous? Could 50 or even 30 be sufficient do we think without risking blocking legit users? Its all a finger in the air anyway with this stuff...

Reply:
It is a bit finger-in-the-airy. I'm happy to lower it to 50; that still feels very comfortable.